Producers

```diff
@@ -390,11 +390,11 @@ type Producer struct {
 	IsActive      int         `json:"is_active"`
 	URL           string      `json:"url"`
 	UnpaidBlocks  int         `json:"unpaid_blocks"`
-	LastClaimTime JSONFloat64 `json:"last_claim_time"`
+	LastClaimTime JSONTime    `json:"last_claim_time"`
 	Location      int         `json:"location"`
 }
 type ProducersResp struct {
-	Producers []Producer `json:"producers"`
+	Producers []Producer `json:"rows"`
```

Comment:
Can you add the remaining missing fields according to the documentation, there is also a `total_producer_vote_weight` which seems to be a `JSONFloat64` and a `more` field which is a `string`.

Reply:
Missing fields are added.
Besides, the field `last_claim_time` is a JSONTime in `nodeos v1.6.1` while it is a Int64 in `nodeos v1.4.4`. So I add a new type `ClaimTime` to fix this problem.